### ISSUE TITLE
Organization of Publications Info Field Is Not Standard

### ISSUE THREAD

**Issue by ericawood** [RTXteam/RTX#1293]
Depending on the source, the publications info dictionary is different. Since SemMedDB came first, the other sources use some elements of its dictionary and not others.

Examples:
- Jensen Lab and RTXKG1 both use the edge object as the key for the sub dictionary (with the score). The sub dictionary has all of the fields, but only object score is not None.
- DrugBank (largely due to my misinterpretation of the standard and the structure of the data) doesn't have a PMID or the edge object as the key for the sub dictionary. Instead, the dictionary is just {"sentence": sentence}
- SemMedDB uses all of the fields listed in the README
- Reactome uses a similar structure to SemMedDB, but only uses the sentence field in the sub dictionary and doesn't feature any of the others

I think it would be helpful if we agreed on a structure that worked for all sources, rather than these hacky and inconsistent methods.

**Comment by saramsey** [comment]
Fair points, thank you for bringing this up. Any suggestions?

**Comment by kvarforl** [comment]
I agree completely! Is the publications info dict something that was homegrown for Semmeddb, or were we attempting to follow a consortium wide standard?

**Comment by saramsey** [comment]
I don't think it is an issue if some sub-components of the structure are missing, for a given knowledge source (if the information is not available). But we should definitely strive for the top-level key to be semantically consistent.

**Comment by saramsey** [comment]
> I agree completely! Is the publications info dict something that was homegrown for Semmeddb, or were we attempting to follow a consortium wide standard?

Homegrown. I believe the EPC standard will ultimately replace it.

**Comment by saramsey** [comment]
Two additional points:
1. EPC working group is addressing this, so we should ultimately see if we can use their schema.
2. In the meantime, we should fix stuff where it is clearly inconsistent with how we did things for SemMedDB, but strive not to make breaking changes unless we are switching wholesale to the new EPC schema (so that downstream consumers of this information don't have to deal with breaking changes twice). Does that make sense?

**Comment by ericawood** [comment]
It seems fairly common that scores are not tied to a particular PMID. Do you all agree on that?

(I've been looking at IntAct and noticed that there. I know that the same is true for JensenLab)

**Comment by kvarforl** [comment]
> It seems fairly common that scores are not tied to a particular PMID. Do you all agree on that?

I agree. The score tends to reflect the confidence of the whole edge in the case of jensen lab, but that's definitely tricky to model with the current setup.

**Comment by kvarforl** [comment]
>     2. In the meantime, we should fix stuff where it is clearly inconsistent with how we did things for SemMedDB, but strive not to make breaking changes unless we are switching wholesale to the new EPC schema (so that downstream consumers of this information don't have to deal with breaking changes twice). Does that make sense?

@ericawood From this I'm interpreting that we should follow this kind of model  https://github.com/RTXteam/RTX/blob/f3e1beddd8034490ba9f8ae8e1bc2ee9f57961b7/code/kg2/jensenlab_tsv_to_kg_json.py#L135-L143 for new sources that we ETL until we get marching orders from EPC.... but not neccesarily make any changes to what's already in KG2? thoughts?
